_includes/header.html

```diff
@@ -12,7 +12,7 @@
   	</div>
   	<div class="collapse navbar-collapse" id="navbar-collapse-1">
   	  <ul class="nav navbar-nav navbar-right">
-  		<li><a href="https://dandiarchive.org/dandiset/drafts">Data Portal</a></li>
+  		<li><a href="https://dandiarchive.org">Data Portal</a></li>
```

Comment:
```suggestion
  		<li><a href="https://dandiarchive.org/portal">Data Portal</a></li>
```

Reply:
I don't follow where this suggestion points to. Are you going to add a new route in the redirector?